README.md

```diff
@@ -1,2 +1,3 @@
-# companion-module-disguise
-See HELP.md and LICENSE
\ No newline at end of file
+# companion-module-generic-osc
```

Comment:
`companion-module-generic-osc` seems to have been added here in error. In fact the repo itself doesn't follow the standard manufacturer-product naming scheme so that's something that may need to be looked at too.

Reply:
Thanks for the input. In re-writing the module I started from scratch with the generic osc module. I missed the README.md still referencing it. 
I agree that the module name could do with updating. Perhaps companion-module-disguise-osc ? 
I'm not in a position to change that though